alchemist-incarnation-scafi/src/main/scala/it/unibo/alchemist/model/implementations/actions/SendScafiMessage.scala

```diff
@@ -30,17 +30,16 @@ class SendScafiMessage[T, P<:Position[P]](
    * useful to support runtime creation of nodes with the same reaction
    * programming, e.g. for morphogenesis.
    *
-   * @param n
+   * @param destinationNode
    * The node where to clone this { @link Action}
    * @param r
    * The reaction to which the CURRENT action is assigned
    * @return the cloned action
    */
-  override def cloneAction(n: Node[T], r: Reaction[T]): Action[T] = {
-    if(!isScafiNode(n)) {
+  override def cloneAction(destinationNode: Node[T], r: Reaction[T]): Action[T] = {
```

Comment:
r -> reaction

Reply:
Ops... I fixed this! Furthermore, while I was at it, I tried to follow clean code guidelines for naming convention in the entire scafi incarnation.. :)